src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/ArchiveGoalTest.kt

```diff
@@ -95,8 +95,8 @@ class ArchiveGoalTest : IntegrationTestBase() {
     await.untilAsserted {
       val timeline = getTimeline(prisonNumber)
       assertThat(timeline)
-        .hasNumberOfEvents(4)
-        .event(4) {
+        .hasNumberOfEvents(5)
+        .event(5) {
```

Comment:
🤔 Has the use of a random prison number introduced a new event somehow? 🤔 
(also, the pendantic sod in me is saying we need to update line 100 also 🤣  - sorry! 🤣 🤷‍♂️ )

Reply:
OK, cool 🤷‍♂️ 🤣